patches/chrome-android-java-AndroidManifest.xml.patch

```diff
@@ -58,23 +58,15 @@ index 6f3722cd64029848c1737ced63889cf7cb41c5aa..266f72ac373c9db6834f4586a88d6234
  
      {% block extra_uses_permissions %}
      {% endblock %}
-@@ -181,6 +194,7 @@ by a child template that "extends" this file.
-     <!-- Set android:largeHeap to "true" to allow more than the default
-          Java heap limit (32Mb on Nexus S, 48Mb on Xoom). -->
-       <application android:name="{% block application_name %}org.chromium.chrome.browser.base.SplitChromeApplication{% endblock %}"
-+        android:requestLegacyExternalStorage="true"
```

Comment:
why is this removed?

Reply:
I've removed it as it was adding this attribute multiple times in many `AndroidStudio.xml` files.
Consequently, the Gradle build generation was failing as each Android Studio XML file was parsed to generate the right build options.
I'm attaching a screenshot of the error message of the original Python script. The error message was not very explanatory, so I ended up adding extra logs and after some digging it turned out that `android:requestLegacyExternalStorage="true"` was the duplicated attribute.

I'm going to check better and see if we can take a safer approach here. (Don't want to risk of leaving out the attribute for some files...)

![image__7_](https://github.com/brave/brave-core/assets/3308503/d2131b98-cb81-48a5-a24a-9555350fa41c)

Reply:
cc @SergeyZhukovsky

Reply:
Also it seems odd to me that this would fail for us when it's part of upstream code. Why does this fail for us, but not for chromium?

Reply:
yeah that's strange, maybe upstream team doesn't generate it or it's not applied for the tests, I think we are ok as far as it's not in the main manifest file.